src/__tests__/databaseTests.js

```diff
@@ -301,6 +301,29 @@ export const matchTests = [
     matching: [{ id: 'm1' }, { id: 'm2', num1: null }, { id: 'm3', num1: 10 }],
     nonMatching: [{ id: 'n1', num1: 0 }, { id: 'n2', num1: 2 }],
   },
+  {
+    name: 'match like (string)',
+    query: [Q.where('text1', Q.like('%ipsum%'))],
+    matching: [
+      { id: 'm1', text1: 'Lorem ipsum dolor sit amet,' },
+      { id: 'm2', text1: 'Lorem Ipsum dolor sit amet,' },
+    ],
+    nonMatching: [
+      { id: 'n1', text1: 'consectetur adipiscing elit.' },
+      { id: 'n2', text1: null },
+    ],
+  },
+  {
+    name: 'match like (null)',
+    query: [Q.where('text1', Q.like(null))],
+    matching: [],
+    nonMatching: [
+      { id: 'n1', text1: 'Lorem ipsum dolor sit amet,' },
+      { id: 'n2', text1: 'Lorem Ipsum dolor sit amet,' },
+      { id: 'n3', text1: 'consectetur adipiscing elit.' },
+      { id: 'n4', text1: null },
+    ],
+  },
```

Comment:
I think we need more test like `%value`, `value%`, `value%value`, `_value`, `value_`, `_value%` to be sure that everything works correctly :)